src/System.ComponentModel.Primitives/src/System/ComponentModel/CategoryAttribute.cs

```diff
@@ -0,0 +1,335 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.ComponentModel
+{
+    /// <devdoc>
+    ///    <para>Specifies the category in which the property or event will be displayed in a
+    ///       visual designer.</para>
+    /// </devdoc>
+    [AttributeUsage(AttributeTargets.All)]
+    public class CategoryAttribute : Attribute, IIsDefaultAttribute
+    {
+        private static volatile CategoryAttribute s_appearance;
+        private static volatile CategoryAttribute s_asynchronous;
+        private static volatile CategoryAttribute s_behavior;
+        private static volatile CategoryAttribute s_data;
+        private static volatile CategoryAttribute s_design;
+        private static volatile CategoryAttribute s_action;
```

Comment:
Nit: should these be in alphabetical order or some other ordering?  Right now the order looks random, but maybe not?